ent/src/yb/tools/yb-admin_client_ent.cc

```diff
@@ -143,19 +144,19 @@ Status ClusterAdminClient::ListSnapshots(bool show_details, bool not_show_restor
   ListSnapshotRestorationsResponsePB rest_resp;
   RETURN_NOT_OK(master_backup_proxy_->ListSnapshotRestorations(rest_req, &rest_resp, &rpc));
 
-  if (not_show_restored) {
-    cout<< "Not show fully RESTORED entries." << endl;
-  }
-
-  if (rest_resp.restorations_size()) {
-    cout << RightPadToUuidWidth("Restoration UUID") << kColumnSep << "State" << endl;
-  } else {
+  if (!rest_resp.restorations_size()) {
```

Comment:
Here I think it's better to write
`if (rest_resp.restorations_size() == 0) {`
(just for better readability)